.github/workflows/monodevelop.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         # Synchronise Mono SSL certs
         cert-sync /etc/ssl/certs/ca-certificates.crt
         # Install extra packages required for dotdevelop build
-        sudo apt-get install -y sed git build-essential intltool nuget fsharp gtk-sharp2
+        sudo apt-get install -y sed git build-essential intltool nuget fsharp gtk-sharp3
```

Comment:
i think there is no need to install gtk-sharp3. 
https://github.com/GtkSharp/GtkSharp/ works on package level.